ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -300,8 +300,8 @@ typeof define === 'function' && define.amd ? define(['exports'], factory) :
       for (const line of layerCoordinates[i]) {
         const linePoints = [];
         for (let j = 0; j < line.length; j += ONE_POINT) {
-          linePoints.push( new THREE.Vector3( line[j], line[j + 1],
-              line[j + 2] + FLOOR_HEIGHT * structureIndex ) );
+          linePoints.push( new THREE.Vector3( line[j],
+              line[j + 2] + FLOOR_HEIGHT * structureIndex, line[j + 1] ) );
```

Comment:
let's make sure to encapsulate soon or else this could become very confusing tech debt :)